### PR TITLE
Obsolete old actions

### PR DESCRIPTION
**Pull request by sonohoshi**
resolve https://github.com/planetarium/lib9c/issues/1411

### CODE REVIEW

File: Lib9c/Action/TransferAsset2.cs [hunk 1]
```diff
@@ -14,10 +14,11 @@ namespace Nekoyume.Action
 {
     /// <summary>
     /// Hard forked at https://github.com/planetarium/lib9c/pull/636
-    /// Updated at https://github.com/planetarium/lib9c/pull/957
+    /// Updated at https://github.com/planetarium/lib9c/pull/1412
     /// </summary>
     [Serializable]
     [ActionType("transfer_asset2")]
+    // Do not add `ActionObsolete` attribute while the Onboarding-portal uses this feature.
```
Comment (boscohyun) [inline Lib9c/Action/TransferAsset2.cs:21]: Can you move this comment above of any attributes? In summary also OK I think.

File: Lib9c/Policy/BlockPolicySource.cs [hunk 2]
```diff
@@ -91,7 +91,7 @@ public partial class BlockPolicySource
         // Target time(KST): 2022. 10. 12 오전 3:42:19
         public const long V100300ObsoleteIndex = 5_150_000;
 
-        public const long V100310ObsoleteIndex = 5_200_400;
+        public const long V100310ObsoleteIndex = 5_416_400;
```
Comment (boscohyun) [inline Lib9c/Policy/BlockPolicySource.cs:94]: Can you comment here what this number mean?

**Review by boscohyun — APPROVED** [review]

File: Lib9c/Action/TransferAsset2.cs (hunk 1 above)
Comment (sonohoshi) [reply Lib9c/Action/TransferAsset2.cs:21]: OK I understood. thanks~

File: Lib9c/Policy/BlockPolicySource.cs (hunk 2 above)
Comment (sonohoshi) [reply Lib9c/Policy/BlockPolicySource.cs:94]: I'll add comment to code. Thx for your opinion!

**Comment by moreal** [comment]
Is this pull request still in work? Or canceled?

**Comment by moreal** [comment]
Maybe this is staled 🤔 If you still need this pull request, please reopen 🙏🏻 @sonohoshi

**Pull request closed without merging**